src/components/auth/login/LoginButton.js

```diff
@@ -7,7 +7,7 @@ import { getDesignRelativeHeight } from '../../../lib/utils/sizes'
 import { isSmallDevice } from '../../../lib/utils/mobileSizeDetect'
 import normalizeText from '../../../lib/utils/normalizeText'
 
-export const LoginButton = ({ style, onPress, testID, icon: Icon, disabled, children, iconStyle }) => {
+export const LoginButton = ({ style, onPress, testID, icon: Icon, disabled, children, iconProps = {} }) => {
```

Comment:
```suggestion
const emptyStyle = {}

export const LoginButton = ({ style, onPress, testID, icon: Icon, disabled, children, iconStyle = emptyStyle }) => {
```

Reply:
Same as below, these should indeed be props and not styles